src/main/java/homework_2/RandomCharsTable.java

```diff
@@ -0,0 +1,76 @@
+package homework_2;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+
+public class RandomCharsTable {
+    public static int strategyFlag;
+    public static StringBuilder stringLetters = new StringBuilder();
+
+
+    public static void main(String[] args) throws IOException {
+        System.out.println("RandomCharTable App");
+        System.out.println("Enter table length, width and strategy separated by a space:");
+
+        try (BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in))){
+            String [] subString = bufferedReader.readLine().split(" ");
+            int tableLength = Integer.parseInt(subString[0]);
+            int tableWidth = Integer.parseInt(subString[1]);
+            String strategy = subString[2];
+            strategyFlag(strategy);
+            tableCharPrinter(tableLength, tableWidth);
+            lettersPrint();
+        }
+    }
+
+    private static void lettersPrint() {
+        if (strategyFlag == 0) {
+            System.out.println("Even letters - " +
+                    stringLetters.replace(stringLetters.length()-2,stringLetters.length()-1, " ") );
```

Comment:
I'm not sure this mechanism works fine. Are you sure that all even/odd letter are printed?